contents/handbook/company/team/charles-cook.md

```diff
@@ -46,7 +45,7 @@ This guide might be helpful in working with me.
 
 ## How you can help me
 
-- If you need something substantive from me that isn't a quick answer, please shoot me an email instead of Slack, so I don't forget it. 
+- If you need me to do something for you (rather than just answer a question), please shoot me an email instead of Slack, as it's easier for me to track. 
```

Comment:
could we make this a github issue?

Reply:
Usually it'll be something more personal that's not well-suited to an issue, so probably not - things like 'can you reimburse me for this personal expense?' or 'can I have an employer reference letter?'.